searx/engines/google_videos.py

```diff
@@ -7,15 +7,16 @@
  @using-api   no
  @results     HTML
  @stable      no
- @parse       url, title, content
+ @parse       url, title, content, thumbnail
 """
 
 from datetime import date, timedelta
 from json import loads
 from lxml import html
+from searx.engines import logger
```

Comment:
Do you mind removing the debug logs?

Reply:
Is it inconvenient for possible future debugging of the functionality? Do I need to remove them completely or is it enough just to comment them out?